pkg/util/util.go

```diff
@@ -83,6 +88,31 @@ func IsResourceIncluded(resource string, backup *velerov1.Backup) bool {
 	return false
 }
 
+func IsResourceExcluded(resource string, backup *velerov1.Backup) bool {
+	if len(backup.Spec.ExcludedResources) == 0 {
+		// Not a "--exclude-resources" backup, assume the resource is included
+		return false
+	}
+
+	for _, res := range backup.Spec.ExcludedResources {
+		if strings.EqualFold(res, resource) {
```

Comment:
this series of 3  equalfold invocations is identical as in the IsResourceIncluded, would be good to have a util extracted

Reply:
:+1: